sdk/lib/3rdparty/freetype/src/psaux/psintrp.c

```diff
@@ -525,7 +525,14 @@
     CF2_ArrStackRec  vStemHintArray;
 
     CF2_HintMaskRec   hintMask;
+
+#ifdef __REACTOS__
+    CF2_GlyphPathRec *glyphPath = malloc(sizeof(CF2_GlyphPathRec));
```

Comment:
I think this function got moved and thus the original ROS diff lost? In freetype_ros.diff we have both this and counterHintMap as heap allocations.

Reply:
Well I booted to shell back then and did several tests, so I happily thought we'd have less diffs now.

Reply:
https://reactos.org/testman/compare.php?ids=57923 suggests we can get away with only glyphPath unless you would like us to get counterHintMap too?